src/main/java/no/uib/cipr/matrix/AbstractMatrix.java

```diff
@@ -60,6 +60,11 @@
  */
 public abstract class AbstractMatrix implements Matrix {
 
+    private static final String A_NUM_COLUMNS_B_NUM_ROWS = "A.numColumns != B.numRows (";
+    private static final String B_NUM_COLUMNS_C_NUM_COLUMNS = "B.numColumns != C.numColumns (";
+    private static final String A_NUM_ROWS_C_NUM_ROWS = "A.numRows != C.numRows (";
+    private static final String A_IS_SQUARE = "!A.isSquare";
```

Comment:
should be `A_IS_NOT_SQUARE`. Likewise for the others.

I can see why the linting tool wants to introduce this refactoring, but I think it results in abstraction that complicates rather than helps.

It might be better to introduce four helper methods that raise the exceptions and take the necessary input parameters.